dynamixel_controllers/src/dynamixel_controllers/calib_required_joint_controller.py

```diff
@@ -92,6 +91,7 @@ def __calib(self):
         if self.torque_limit is not None:
             self.set_torque_limit(self.torque_limit)
         motor_states_sub_for_init.unregister()
+        init_pos = self.motor_states_for_init['position']
```

Comment:
This line should be before `self.set_torque_enable(False)` at least because `self.set_torque_enable(False)` makes motor torque off.
If init_pos is taken after that, init_pos may be different from limit position.
And `self.set_torque_enable(False)` cannot be deleted for my gripper because keeping torque on around limit makes the timing belt slipped.
I'm not sure whether this line should be after `self.__set_speed_wheel(0.0)`.
If you can find another way to fix, I'm happy.